pt-BR/routing.md

```diff
@@ -580,153 +580,153 @@ Isto vai permitir que o Rails reconheça _paths_ como `/comments/new/preview` co
 
 TIP: Se você se encontrar adicionando muitas _actions_ extras para uma rota _resourceful_, é um bom momento para perguntar a si mesmo se você está mascarando a presença de outro _resource_.
 
-Non-Resourceful Routes
+Rotas sem uso de *Resources*
```

Comment:
Achei essa tradução bem boa, ficou bem semântica.

Reply:
O que acham de mudarmos a tradução da página toda para manter `resource` e na primeira colocar algo como: *Resource (Recurso)*?

@campuscode/guia-rails-team

Reply:
Em outros lugares dessa página usamos recurso então coloquei `Recurso (Resources)` o que acha? Essa troca teria que ser feita em outros lugares no futuro.

Reply:
> Em outros lugares dessa página usamos recurso então coloquei `Recurso (Resources)` o que acha? Essa troca teria que ser feita em outros lugares no futuro.

Depois vai ter que rolar uma força tarefa pra sair equalizando todos os termos e ir incrementando no nosso manual.

Reply:
Sim 😞